### ISSUE TITLE
Last 5 logged-in users list is ambiqous

### ISSUE THREAD

**Issue by N6REJ** [joomla/joomla-cms#38820]
### Steps to reproduce the issue
Have last 5 logged-in user module in admin active.
log in with more then 2 users

### Expected result
lists historical last x logged in users ( up to limit ) 

### Actual result
![image](https://user-images.githubusercontent.com/1850089/191896528-df8c439f-f164-4ed7-aada-f25b74b1355b.png)

### System information (as much as possible)
[systeminfo-2022-09-23T05_23_28+00_00.txt](https://github.com/joomla/joomla-cms/files/9630881/systeminfo-2022-09-23T05_23_28%2B00_00.txt)

### Additional comments
As shown in our log, there were more then 2 "last logged in users"
![image](https://user-images.githubusercontent.com/1850089/191896615-83c0219c-b064-4c92-8b03-4572e89fba81.png)
![image](https://user-images.githubusercontent.com/1850089/191896640-2fdf8239-f877-45c8-9756-a4cba403973a.png)

**Comment by N6REJ** [comment]
@joomla/ux-team

**Comment by alikon** [comment]
it depends from session life time, what is your sessions life time ?

**Comment by brianteeman** [comment]
It is "last five logged in users who still have open sessions"

**Comment by N6REJ** [comment]
> It is "last five logged in users who still have open sessions"

Right, which is where the ambiguity comes in.  The verbiage states that its the last 5 users logged in.  When infact that is not the case.

**Comment by brianteeman** [comment]
feel free to come up with an alternative suggestion or just change the module title on your own site.

**Comment by N6REJ** [comment]
Would be nicer to change the function.  So it really is the last X users
